protocol/scripts/create_full_node.sh

```diff
@@ -0,0 +1,113 @@
+# Add random number to the end of name to keep a unique name for the node
+NODE_NAME="FullNodeMainnet-$RANDOM" 
+
+# Update system and install dependencies
+
+sudo apt-get -y update
+sudo apt-get install -y curl jq lz4
+
+
+# Get architecture
+if [ $(uname -m) = "aarch64" ]; then
+ARCH=arm64
+elif [ $(uname -m) = "x86_64" ]; then
+ARCH=amd64
+else
+echo "This device is not arm64 or amd64, please use a device with arm64 or amd64 architecture"
+fi
+
+# Install go
+wget https://golang.org/dl/go1.22.2.linux-$ARCH.tar.gz 
+sudo tar -C /usr/local -xzf go1.22.2.linux-$ARCH.tar.gz
+rm go1.22.2.linux-$ARCH.tar.gz
+echo 'export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin' >> $HOME/.bashrc
+eval "$(cat $HOME/.bashrc | tail -n +10)"
+
+# Install Cosmovisor
+go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.5.0
+
+# Download dydxprotocold binaries and initialize node
+mkdir -p $WORKDIR
+cd $WORKDIR
+FILE_REGEX="dydxprotocold-v[0-9]+\.[0-9]+\.[0-9]+-linux-$ARCH.tar.gz"
+curl -s "https://api.github.com/repos/dydxprotocol/v4-chain/releases/tags/protocol/$VERSION" \
+| grep -E $FILE_REGEX \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
+tar -xvf dydxprotocold-v*-linux-$ARCH.tar.gz && rm dydxprotocold-v*-linux-$ARCH.tar.gz
+cd $WORKDIR/build
+mv dydxprotocold-*-linux-$ARCH dydxprotocold
+chmod +x dydxprotocold
+./dydxprotocold init --chain-id=$CHAIN_ID $NODE_NAME
+
+# Create cosmovisor directories and move binaries
+mkdir -p $HOME/.dydxprotocol/cosmovisor/genesis/bin
+mkdir -p $HOME/.dydxprotocol/cosmovisor/upgrades
+mv dydxprotocold $HOME/.dydxprotocol/cosmovisor/genesis/bin/
+
+# Update config
+curl https://dydx-rpc.lavenderfive.com/genesis | python3 -c 'import json,sys;print(json.dumps(json.load(sys.stdin)["result"]["genesis"], indent=2))' > $WORKDIR/config/genesis.json
+sed -i 's/seeds = ""/seeds = "'"${SEED_NODES[*]}"'"/' $WORKDIR/config/config.toml
+
+# Create Service
+sudo tee /etc/systemd/system/dydxprotocold.service > /dev/null << EOF
+[Unit]
+Description=osmosis node service
+After=network-online.target
+
+[Service]
+User=$USER
+ExecStart=/$HOME/go/bin/cosmovisor run start --non-validating-full-node=true
+WorkingDirectory=$HOME/.dydxprotocol
+Restart=always
+RestartSec=5
+LimitNOFILE=4096
+Environment="DAEMON_HOME=$HOME/.dydxprotocol"
+Environment="DAEMON_NAME=dydxprotocold"
+Environment="DAEMON_ALLOW_DOWNLOAD_BINARIES=false"
+Environment="DAEMON_RESTART_AFTER_UPGRADE=true"
+Environment="UNSAFE_SKIP_BACKUP=true"
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo systemctl daemon-reload
+sudo systemctl enable dydxprotocold
+
+
+# Get snapshot
+cd $WORKDIR
+xml_content=$(curl -s "${BASE_SNAPSHOT_URL}")
+file_key=$(echo "$xml_content" | grep -oP '<Key>dydx/dydx_\d+\.tar\.lz4</Key>' | head -1 | sed 's/<[^>]*>//g')
+file_url="${BASE_SNAPSHOT_URL}${file_key}"
+wget -O "snapshot.tar.lz4" "${file_url}"
+lz4 -c -d snapshot.tar.lz4 | tar -x -C $WORKDIR
+rm snapshot.tar.lz4
+
+# Start service
+sudo systemctl start dydxprotocold
```

Comment:
either:
1. keep this and rename `create_and_run_full_node.sh`
2. do not start service, put out a message on how to run

either way I would put out a message at the end indicating how to start/stop the node, and indicate installation completed